### PR TITLE
Fix database clearing

### LINKED ISSUE #430: remove beacon network stats on removing it

### LINKED ISSUE #434: remove valdiator balance history on removing validator

### PR DESCRIPTION
**Pull request by BeroBurny**
closes #434 
closes #430

### CODE REVIEW

File: src/main/logger/index.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ export const cgLogger = createLogger("chainGuardian", "chainGuardian.log");
 
 export const getBeaconLogfileFromURL = (host: string): string => {
     const url = new URL(host);
-    let sanitizedName = (url.host + url.pathname).replace(/[^a-zA-Z0-9:]/g, "_");
+    let sanitizedName = (url.host + url.pathname).replace(/[^a-zA-Z0-9]/g, "_");
```
Comment (mpetrunic) [inline src/main/logger/index.ts:18]: can you add comment what characters are offending and why?

File: src/renderer/ducks/validator/sagas.ts [hunk 2]
```diff
@@ -139,13 +139,16 @@ export function* addNewValidatorSaga(action: ReturnType<typeof addNewValidator>)
...
     cgLogger.info("Removing validator", action.payload);
+    yield put(unsubscribeToBlockListening(action.payload));
+
     const auth: CGAccount | null = yield select(getAuthAccount);
     deleteKeystore(auth.directory, action.payload);
     auth.removeValidator(action.meta);
 
-    yield put(unsubscribeToBlockListening(action.payload));
+    yield database.validator.balance.delete(action.payload);
+    yield database.validator.beaconNodes.delete(action.payload);
```
Comment (mpetrunic) [inline src/renderer/ducks/validator/sagas.ts:151]: yield all([])?

Comment (BeroBurny) [reply src/renderer/ducks/validator/sagas.ts:151]: i do not get it

Comment (mpetrunic) [reply src/renderer/ducks/validator/sagas.ts:151]: so it runs in parallel:
```
yield all([
   database.validator.balance.delete(action.payload),
   database.validator.beaconNodes.delete(action.payload)
])
```

**Review by mpetrunic — APPROVED** [review]

**Pull request merged**